source/jormungandr/jormungandr/scenarios/helper_classes/fallback_durations.py

```diff
@@ -117,12 +117,22 @@ def _do_request(self):
                 return {center_isochrone.uri: DurationElement(0, response_pb2.reached)}
             else:
                 return result
-        sn_routing_matrix = self._instance.get_street_network_routing_matrix([center_isochrone],
-                                                                             places_isochrone,
-                                                                             self._mode,
-                                                                             self._max_duration_to_pt,
-                                                                             self._request,
-                                                                             **self._speed_switcher)
+
+        if self._origin:
```

Comment:
I think we can get this _origin another way, I don't remember how, come and we'll try to search for it